doc/flame/components.md

```diff
@@ -1104,6 +1104,18 @@ Check the example app
 for details on how to use it.
 
 
+## RowComponent & ColumnComponent
```

Comment:
if it is experimental, should we flag it in the doc, similar to how it is done [elsewhere](https://github.com/flame-engine/flame/blob/47970224f8c9c90718c54301ee69d9cddcced87b/doc/flame/camera_component.md)?